lang/java/avro/src/test/java/org/apache/avro/TestDataFileReader.java

```diff
@@ -38,9 +38,12 @@
 import org.apache.avro.generic.GenericDatumReader;
 import org.apache.avro.generic.GenericDatumWriter;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 @SuppressWarnings("restriction")
 public class TestDataFileReader {
+  @TempDir
+  public Path DATA_DIR;
```

Comment:
Can we relook at the variable name once please? This format used is for constants, giving an impression that it is static final.